api/controllers/posts.js

```diff
@@ -22,6 +22,23 @@ const PostsController = {
       res.status(201).json({ message: 'OK', token: token });
     });
   },
-};
+  Update: (req, res) => {
+    const updated_post = req.body.message;
+    let post_id = req.params.id;
+    Post.findByIdAndUpdate(post_id, {"message": updated_post}, { new: true},
+    (err, post) => {
+      if (err) {
+        throw err;
```

Comment:
Instead of using `throw` here, it might be better to return a 500 error to the client, and log the error — throwing will cause the entire server to crash, and because we're not checking first which kind of error it is, the program will crash no matter what, even if it's an error the program can safely handle. If the server crashes because of using `throw`, it will cause it to fail for all other users trying to use the app!

Reply:
Thank you for the feedback. This is now changed. The pull request will be updated, once we have implemented the authorization of the user editing the post.